docs/reference/aggregations/pipeline/movavg-aggregation.asciidoc.orig

```diff
@@ -0,0 +1,666 @@
+[[search-aggregations-pipeline-movavg-aggregation]]
```

Comment:
I don't think you meant to include this file? This looks like leftover from a merge conflict?